libraries/root_generic/macosx_math.cpp

```diff
@@ -206,7 +206,7 @@ float tanf( float v )
 
 float powf( float v, float p )
 {
-    return (float) pow( (double) v, (double) p );
+    return (float) std::pow( (double) v, (double) p );
```

Comment:
Do we really still need macos specific files?

Reply:
No, no we do not. This should not really be made macOS-specific. Including <cmath> and then using `std::pow` is the way to do it, everywhere, across the board. Let's just do that.

Reply:
So this whole function would go away, wouldn't it?

Reply:
Sure looks that way.

Reply:
OK. Can that be done in a separate PR @royfalk ?

Reply:
Of course.